content/installation/net/install/NetInstallation.md

```diff
@@ -29,33 +29,23 @@ You can fully configure the agent using the *contrast_security.yaml* file. See t
 
-## Silent Installation
+## Silent and Custom Installations
 
-The .NET agent installer supports following the command line options. These options are silent, which means that they don't require your interaction and don't present the installer's UI.
-
-* Install: `ContrastSetup.exe -s -norestart`
-
-* Install and do not start the Contrast .NET Tray Application: `ContrastSetup.exe -s -norestart StartTray=0`
-
-* Install using the YAML file specified: `ContrastSetup.exe -s PathToYaml=c:\contrast_security.yaml`
-
-* Uninstall: `ContrastSetup.exe -uninstall -s -norestart`
-
-* Repair: `ContrastSetup.exe -s -repair`
+The .NET agent installer can be customized using command line options. See the [installer command line options](TODO Link) for more information.
 
 ## Changes Made by the Agent Installer
 
 Many users are curious about the changes made by the .NET agent and the impacts these changes may have. In all respects, the Contrast .NET agent installer is a normal Windows application installer built using standard MSI technology. The .NET agent installer validates that the target server satisfies several requirements (e.g., the operating system is Windows Server 2008 R2 or greater). If all requirements are met, the installer registers the .NET agent as a standard Windows program and makes the following changes.
 
-- Places the agent’s files on a disk in the specified install location (e.g., *C:\Program Files\Contrast\dotnet*). This includes several dynamic link libraries (DLLs) and executables, such as the background Windows service that drives agent behavior. 
-- Creates the specified data directory for the agent that's primarily used to store agent log files and configuration (e.g., *C:\ProgramData\Contrast\dotnet*). 
+- Places the agent’s files on a disk in the specified install location (e.g., *C:\Program Files\Contrast\dotnet*). This includes several dynamic link libraries (DLLs) and executables, such as the background Windows service that drives agent behavior.
+- Creates the specified data directory for the agent that's primarily used to store agent log files and configuration (e.g., *C:\ProgramData\Contrast\dotnet*).
 - Registers the agent’s background Window service with the operating system.
-- Starts the agent’s background Windows service and Tray (UI) application. This service has a number of responsibilities: 
-  - Preparing the environment for instrumentation by registering the agent’s profiler component with IIS through environment variables, and restarting IIS. This causes the CLR to load the agent’s profiler, which is responsible for instrumenting analyzed applications. 
+- Starts the agent’s background Windows service and Tray (UI) application. This service has a number of responsibilities:
```

Comment:
Should we point out the background service name here?  It might be worth mentioning that the agent can be restarted by restarting the service: this is something we've recommended to people for a while as a foolproof way to restart the agent.  Not sure if this the best place for it but it seems like this document is going beyound 'installation' and into 'system documentation'.

Reply:
I really want to update the agent service name before documenting it. 😹 

We could recommend using the those shortcuts to restart the service instead? I'm not sure if those are documented anywhere.